index.js

```diff
@@ -55,7 +55,7 @@ Merry.prototype.router = function (opts, routes) {
       // val should ideally be a stream already, but if it's not we got you bae
       handler(req, res, ctx, function (err, val) {
         if (err) {
-          res.statusCode = err.statusCode || res.statusCode || 500
+          res.statusCode = err.statusCode || 500
```

Comment:
I see what you're saying, but is there a typo in your suggestion? I think what you want is:
```js
res.statusCode = err.statusCode || (res.statusCode >= 400 ? res.statusCode : 500)
```

`err.statusCode` should always take precedence (merry.error style), then allow any `res.statusCode` from the error range (or maybe any non-200 `res.statusCode`), then fall back to 500